apps/web/src/components/ui/carousel.tsx

```
@@ -1,241 +1,241 @@
+				orientation:
+					orientation || (opts?.axis === "y" ? "vertical" : "horizontal"),
+				scrollPrev,
+				scrollNext,
+				canScrollPrev,
+				canScrollNext,
+			}}
+		>
+			<div
+				onKeyDownCapture={handleKeyDown}
+				className={cn("relative", className)}
+				role="region"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)